src/modules/inventing/QuickInvent.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
+  import daDoInvent from '../_dataAccess/daDoInvent';
+  import { sendEvent } from '../support/fshGa';
+
+  export let max;
+  export let recipeID;
+  let amountToInvent = 1;
+  let invResults = '';
+  let results = [];
+
+  function maxInvent() {
+    amountToInvent = max;
+    sendEvent('inventing', 'maxInventButton');
+  }
+
+  async function quickInvent() {
+    sendEvent('inventing', 'quickInvent');
+    results = [];
+    invResults = '';
+    if (!amountToInvent) { return; }
+    invResults = `Inventing ${amountToInvent} Items`;
+    const requests = Array(amountToInvent).fill(recipeID);
+    requests.reduce(async (prev, recipe) => {
+      await prev;
+      return daDoInvent(recipe).then((json) => {
+        results = [...results, json];
+        if (json.e.message) { throw Error('Inventing failure'); }
```

Comment:
where are you catching this error?
is this an attempt at short circuiting the reduce?
I recently did something similar with ```src/modules/trade/oneByOne.js```

I am concerned here that if I run out of components it will continue to attempt invention causing multiple network round trips.